### ISSUE TITLE
Is it possible to share the F1 score or accuracy of pre-training the models in their control setting in the image sentence alignment objective?

### ISSUE THREAD

**Issue by kaiweicen** [e-bug/volta#21]
Hi, thanks to the great repository! Is it possible to post the F1 score or accuracy of pre-training the models in their control setting in the image sentence alignment objective? I don't see these information in the VOLTA paper.

**Comment by e-bug** [comment]
Hi,

Can you clarify what is your goal? 

Also, you can run any analysis on _all_ the models pretrained in the control setup (cf. [here](https://github.com/e-bug/volta/blob/main/MODELS.md#checkpoints-by-random-seed))
